src/CoreConnector.ts

```diff
@@ -20,21 +21,23 @@ export class CoreEventManager {
   constructor(private connector: BaseConnector) {}
 
   public addEvent(
-    eventOptions: any,
+    eventOptions: Options,
     handler: CoreEventHandler,
     eventId: string | Record<string, any>,
   ): EventConfiguration {
     const id =
       typeof eventId == 'string'
```

Comment:
Shouldn't we use `===` ?

Reply:
I think this was a typo in the original code. Also fixed in CoreConnector.onStart